packages/react-client/src/index.ts

```diff
@@ -13,24 +13,7 @@ export {
 } from "./hooks/public";
 export { FishjamProvider } from "./fishjamProvider";
 
-export type { PeerState, Track, PeerId, TrackId, TrackWithOrigin, Origin, PeerStatus } from "./state.types";
-
-export type {
-  DeviceManagerConfig,
-  StorageConfig,
-  Devices,
-  Device,
-  AudioDevice,
-  UseSetupMediaResult,
-  UseSetupMediaConfig,
-  ScreenshareApi,
-  UseConnect,
-  ConnectConfig,
-  TrackMiddleware,
-  PeerStateWithTracks,
-  TrackMetadata, // only for compatibility reasons, will be removed in: FCE-415
-  PeerMetadata, // only for compatibility reasons, will be removed in: FCE-415
-} from "./types";
+export * from "./types/public";
```

Comment:
I'm not sure if this approach is better than old one.
With old one approach, you need to add type to export list explicit. Right now, any type added to public.ts will be 'public'. And it might happen by accident, as people tend to not read exact file names sometimes.

Reply:
we might as well do it explicitly